src/java_bytecode/java_bytecode_instrument.cpp

```diff
@@ -13,6 +13,7 @@ Date:   June 2017
 #include <util/std_code.h>
 #include <util/std_expr.h>
 #include <util/symbol_table.h>
+#include <util/c_types.h>
```

Comment:
Including `c_types` in a Java specific module looks a bit suspicious.

Reply:
True; we should probably rename `c_types` to `platform_types` as by now it already does include Java-specific types. The key property of the type names in that file is that they depend on what `config` has...

Reply:
yes, I also like the idea of having 'platform_types', and then perhaps have the really C-specific ones in ansi-c/c_types.h (where that file used to live).